lib/chef/knife.rb

```diff
@@ -640,7 +640,7 @@ def test_mandatory_field(field, fieldname)
     def rest
       @rest ||= begin
         require_relative "server_api"
-        Chef::ServerAPI.new(Chef::Config[:chef_server_url])
+        Chef::ServerAPI.new(Chef::Config[:chef_server_root])
```

Comment:
doesn't this change the rest API for all the rest of the knife commands though?

i was okay with the root_rest object but just didn't want it v1.